ztools/ztls/common.go

```diff
@@ -352,6 +352,10 @@ type Config struct {
 
 	// Explicitly set Client random
 	ClientRandom []byte
+
+	// Explicitly set ClientHello
+	ForceClientHello bool
```

Comment:
I think we can get rid of this bool, and just use`ExternalClientHello != nil` as an indicator that we have an external client hello?